pegasus/sites.v3/hourofcode.com/public/leaderboards.haml

```diff
@@ -0,0 +1,62 @@
+---
+title: Leaderboards for the Hour of Code
+theme: responsive
+
+social:
+  'og:image' : "http://csedweek.org/images/girls-58-boys-42.jpg"
+  'og:title' : 'Who has done the Hour of Code?'
+  'og:description' : "Over 400 million Hours of Code have been served in 180+ countries and 45+ languages. Anyone can learn an hour of computer science."
+  "twitter:card" : 'photo'
+  "twitter:site" : "@codeorg"
+  "twitter:title" : "Who has done the Hour of Code?"
+  "twitter:image:src" : "http://csedweek.org/images/girls-58-boys-42.jpg"
+  "twitter:image:width" : 759
+  "twitter:image:height" : 208
+---
+-twitter_message = I18n.t('who_has_done_hoc_donor', donor_twitter: PegasusCdoDonor.get_random_donor_twitter)
+-facebook = {:u=>"https://www.facebook.com/photo.php?fbid=521364791293162"}
+-twitter = {:url=>"http://#{request.site}/leaderboards", :related=>'codeorg', :hashtags=>'', :text=>twitter_message}
+-twitter[:hashtags] = 'HourOfCode' unless twitter_message.include? '#HourOfCode'
+
+-hoc_metrics = fetch_hoc_metrics
+-started = format_integer_with_commas(hoc_metrics['started'])
+
+-loc = format_integer_with_commas(fetch_metrics['lines_of_code'])
+
+#congrats.mobile-pad{:style=>'margin: 0 auto;'}
+  %h1{:style=>'margin-bottom:10px'} The Hour of Code Leaderboards
+  %div{:style=>'font-size:18px'}
+    %b= "Total participation: #{started} served. Students have written #{loc} lines of code."
+
+  = view :votes, results:hoc_metrics
```

Comment:
This partial needs to be moved to `pegasus/sites.v3/hourofcode.com/views` — getting this error:

<img width="1102" alt="Screenshot 2023-09-27 at 10 17 08 AM" src="https://github.com/code-dot-org/code-dot-org/assets/9256643/bc917e6c-3e8e-4884-bfcd-6f789037059c">